protocol/pom.xml

```diff
@@ -120,6 +120,7 @@
                 <configuration>
                   <cleanOutputFolder>false</cleanOutputFolder>
                   <outputDirectory>${basedir}/src/main/java</outputDirectory>
+                  <ignoreVersions>true</ignoreVersions>
```

Comment:
* needed because there is a CI job that uses an older protobuf lib than our protobuf-java lib
* I tried to get the CI job to use a newer protobuf lib but the format of the protobuf zip has changed and I'm not sure how to get the existing build to use a newer protobuf zip